app/connectors/v3/getPenaltyDetails/GetPenaltyDetailsConnector.scala

```diff
@@ -27,7 +30,21 @@ class GetPenaltyDetailsConnector @Inject()(httpClient: HttpClient,
                                         appConfig: AppConfig)
                                        (implicit ec: ExecutionContext) {
 
-  def getPenaltyDetails(vatcUrl: String)(implicit hc: HeaderCarrier): Future[GetPenaltyDetailsResponse] = {
-    httpClient.GET[GetPenaltyDetailsResponse](url = appConfig.getPenaltyDetailsUrl + vatcUrl)
+  implicit val hc: HeaderCarrier = headersForEIS
+
+  def getPenaltyDetails(vrn:String): Future[GetPenaltyDetailsResponse] = {
+    httpClient.GET[GetPenaltyDetailsResponse](url = appConfig.getPenaltyDetailsUrl + vrn)
+  }
+
+  def headersForEIS: HeaderCarrier = {
+    val headers = Seq(
+      "Environment"      -> appConfig.eisEnvironment,
+      ACCEPT             -> MimeTypes.JSON,
+      AUTHORIZATION -> s"Bearer ${appConfig.eiOutboundBearerToken}"
```

Comment:
I believe it is required for all EIS calls even though it is not in the spec, So I thought best to include just in case,The appeals Spec doesn't show it either